jupyterhub/traitlets.py

```diff
@@ -125,23 +132,23 @@ def help(self):
         chunks = [self._original_help]
         chunks.append("Currently installed: ")
         for key, entry_point in self.load_entry_points().items():
-            chunks.append(
-                "  - {}: {}.{}".format(
-                    key, entry_point.module_name, entry_point.object_name
-                )
-            )
+            chunks.append(f"  - {key}: {entry_point.module}.{entry_point.attr}")
         return '\n'.join(chunks)
 
     @help.setter
     def help(self, value):
         self._original_help = value
 
     def load_entry_points(self):
-        """Load my entry point group"""
-        # load the group
-        group = entrypoints.get_group_named(self.entry_point_group)
-        # make it case-insensitive
-        return {key.lower(): value for key, value in group.items()}
+        """Load my entry point group
+
+        Returns a dict whose keys are lowercase entrypoint names
+        """
+        group = entry_points(group=self.entry_point_group)
+        return {
+            entry_point.name.lower(): entry_point
+            for entry_point in entry_points(group=self.entry_point_group)
```

Comment:
```suggestion
            for entry_point in group
```

Reply:
Maybe the name `group` could be `entry_points` or `group_entry_points` or similar.